construct-binary-tree-from-preorder-and-inorder-traversal/delight010.swift

```diff
@@ -0,0 +1,26 @@
+class Solution {
+    // Time O(N)
```

Comment:
알고달레 [풀이 1](https://www.algodale.com/problems/construct-binary-tree-from-preorder-and-inorder-traversal/#%ED%92%80%EC%9D%B4-1)의 `index`처럼 `for` 반복문이 병목이 됩니다. 선형 시간으로 줄일 수는 있던데 좀 어렵더라고요!
```suggestion
    // Time O(N^2)
```